students/yiju/users/views.py

```diff
@@ -47,14 +51,18 @@ def post(self,request):
 			data     = json.loads(request.body)
 			email    = data['email']
 			password = data['password']
+			user     = User.objects.get(email=email)
 
 			if not User.objects.filter(email=email).exists():
 				return JsonResponse({'message':'INVALID_USER'}, status=401)
 
-			if not password == User.objects.get(email=email).password:
+			if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
 				return JsonResponse({'message':'INVALID_USER'}, status=401)
 			
-			return JsonResponse({'message':'SUCCESS'}, status = 200)
+			SECRET = SECRET_KEY
+			token  = jwt.encode({'id' : user.id }, SECRET, algorithm = 'HS256')
```

Comment:
```suggestion
			token  = jwt.encode({'id' : user.id }, SECRET_KEY, algorithm = 'HS256')
```
로직을 파악하고 계시니 변수화 없이 바로 쓰는게 더 깔끔하겠죠?